samples/sample_common/src/vaapi_utils.cpp

```diff
@@ -451,7 +451,6 @@ void CLibVA::ReleaseVASurface(
         AcquireCtx* ctx = (AcquireCtx*)actx;
         if (ctx) {
             m_libva.vaDestroySurfaces(dpy2, &srf2, 1);
-            close(ctx->fd);
```

Comment:
When is this closed?

Reply:
> When is this closed?
 AcquireVASurface is just handle surface thing, No fd is exposed to outside, so we don't need to close it.

Reply:
https://github.com/Intel-Media-SDK/MediaSDK/issues/2983

Reply:
Prefer below change:
if (ctx->fd) {
close(ctx->fd);}

Reply:
sure.  I updated it based on your feedback.

Reply:
Thanks for the change.